atlasdb-config/build.gradle

```diff
@@ -37,3 +46,17 @@ dependencies {
     testCompile group: 'com.github.tomakehurst', name: 'wiremock'
     testCompile group: 'org.assertj', name: 'assertj-core'
 }
+
+shadowJar {
+    mergeServiceFiles()
+    classifier ''
+
+    relocate('com.palantir.remoting2.errors', 'com.palantir.atlasdb.http.errors')
```

Comment:
Sure, changed. Yep, this makes sense - it's consistent with how we do it for Dagger as well.